pages/index.js

```diff
@@ -17,20 +19,24 @@ export default function HomePage({ activities, onToggleBookmark }) {
   );
 
   return (
-    <section>
-      <StyledAddLink href="/createActivity">+ New</StyledAddLink>
+    <StyledSection>
+      <Link href="/createActivity">
```

Comment:
For the Link is the same as for the button. Screenreader needs an aria-label to identify the purpose of the link.